apim/3.x/templates/api/api-configmap.yaml

```diff
@@ -457,6 +457,16 @@ data:
         enabled: true
         ws:
           discovery: true
+          sendEventsOnHttp: {{ .Values.alerts.sendEventsOnHttp | default true }}
```

Comment:
I would prefer a block of code to handle this part. 
It's more flexible.
WDYT?